lib/build/web_components.dart

```diff
@@ -25,7 +25,7 @@ Asset generateWebComponentsBootstrap(Resolver resolver, Transform transform,
     dom.Document document, AssetId scriptId, AssetId newScriptId,
     {List<InitializerPlugin> extraPlugins: const []}) {
   var htmlImportRecorder = new HtmlImportAnnotationRecorder();
-  var plugins = [htmlImportRecorder]..addAll(extraPlugins);
+  var plugins = [htmlImportRecorder as InitializerPlugin]..addAll(extraPlugins);
```

Comment:
Yeah, I think `<InitializerPlugin>[htmlImportRecorder]` is the cleanest solution here.  If you don't put in the explicit type, strong mode infers it (for list and map literals) from the contents.  In this case, it appears that is too strong - you want to add different plugins after.